apps/app/src/pages/Owner/GeneralSettings.js

```diff
@@ -106,41 +119,143 @@ function PlanCard({ purchases, screenshotsLimitPerMonth, ...props }) {
+    0
+  );
+
+  return (
+    <x.div mt={2}>
+      <x.div
+        display="flex"
+        justifyContent="space-between"
+        alignItems="baseline"
+      >
+        <x.div>
+          {screenshotsCount.toLocaleString()}{" "}
+          <x.span> {pluralize("screenshot", screenshotsCount)}</x.span>
+        </x.div>
+        <x.div color="secondary-text"> / {total.toLocaleString()}</x.div>
+      </x.div>
+
+      <ProgressBar score={screenshotsCount} total={total} updateColor mt={2} />
+
+      {repositories.length > 0 ? (
+        <RepositoriesDetailsConsumption repositories={repositories} />
+      ) : null}
+    </x.div>
+  );
+}
+
+function UsageCard({ screenshotsLimitPerMonth, repositories, ...props }) {
+  const publicActiveRepos = repositories.filter(
+    (repo) => !repo.private && repo.currentMonthUsedScreenshots > 0
+  );
+
+  const privateActiveRepos = repositories.filter(
+    (repo) => repo.private && repo.currentMonthUsedScreenshots > 0
+  );
+
   return (
     <Card {...props}>
       <CardHeader>
-        <CardTitle>Usage</CardTitle>
+        <CardTitle>Monthly usage</CardTitle>
       </CardHeader>
-      <CardBody>
-        <CardTitle mb={2}>Used screenshots</CardTitle>
-        <x.span fontSize="xl">
-          {currentMonthUsedScreenshots.toLocaleString()}
-        </x.span>{" "}
-        <x.span color="gray-500">/ {screenshotsLimitPerMonth}</x.span>
-        <ProgressBar
-          score={currentMonthUsedScreenshots}
-          total={screenshotsLimitPerMonth}
-          updateColor
-          mt={2}
-        />
-        <CardTitle mt={4} mb={2}>
-          Next reset date
-        </CardTitle>
-        <Time date={moment().add(1, "month").startOf("month").toISOString()} />
+      <CardBody display="flex" gap={8} flexDirection="column">
+        <div>
+          <CardTitle>Public repositories</CardTitle>
+          <RepositoriesConsumption
+            repositories={publicActiveRepos}
+            total={Infinity}
+          />
+        </div>
+
+        <div>
+          <CardTitle>Private repositories</CardTitle>
+          <RepositoriesConsumption
+            repositories={privateActiveRepos}
+            total={screenshotsLimitPerMonth}
+          />
+        </div>
+
+        <x.div
+          display="flex"
+          alignItems="baseline"
+          flexWrap="wrap"
+          columnGap={4}
+        >
+          <CardTitle whiteSpace="nowrap">Next reset date</CardTitle>
+          <Time
+            date={moment().add(1, "month").startOf("month").toISOString()}
+            flex={1}
+            textAlign="right"
+            whiteSpace="nowrap"
+          />
+        </x.div>
       </CardBody>
     </Card>
   );
 }
 
 export function GeneralSettings({
-  owner: { name, purchases, currentMonthUsedScreenshots },
+  owner: { name, purchases, screenshotsLimitPerMonth, repositories },
 }) {
-  const screenshotsLimitPerMonth = Infinity;
+  const minScreenshotLimit = screenshotsLimitPerMonth || 7000;
```

Comment:
hmm ?

Reply:
They are 2 interesting topic here :
- the ˋ|| 7000ˋ is here for people without plan purchase. By default repo have 7000 private screenshot / month even without plan. 

- the second topic is that I have here replaced `infinity` by a the plan's screenshots imite. It's maybe too early ?

Reply:
Actually it id unlimited for every one, so please remove this fake limit.